src/main/java/com/iota/iri/service/tipselection/impl/WalkerAlpha.java

```diff
@@ -128,7 +128,7 @@ public Hash walk(Hash entryPoint, Map<Hash, Integer> ratings, WalkValidator walk
         int approverIndex;
 
         //Check if ratings map is empty. If so, alpha was set to 0 and a random approver will be selected.
```

Comment:
Youre probably going to stab me once or twice, but the comment above it is now wrong :)